frontendservice/src/views/specificShop.vue

```diff
@@ -0,0 +1,139 @@
+/* Copyright 2020 Google LLC Licensed under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance with the License. You
+may obtain a copy of the License at https://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed
+under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License. */
+
+<template>
+  <div>
+    <NavBarCustomer></NavBarCustomer>
+    <cookieinfo></cookieinfo>
+    <cookieackno></cookieackno>
+    <div class="card">
+      <div class="card-content">
+        <p class="title">{{shopinfo.shop.shopName}}</p>
+        <p v-if="flag" class="title">Number of People ahead in the queue are:</p>
+        <p
```

Comment:
Nit. Code in lines 18 to 21 can be in the same line maybe.

Reply:
Same here happening automatically when I save the code in vscode